automation/src/test/scala/org/broadinstitute/dsde/test/api/MultiCloudWorkspaceSpec.scala

```diff
@@ -31,4 +31,4 @@ class MultiCloudWorkspaceSpec extends AnyFreeSpecLike {
       thrown.fields("statusCode").convertTo[Int] shouldBe 501
     }
   }
```

Comment:
Some point soon I imagine we want to add an integration test that we can actually create an Azure workspace from rawls.

Reply:
@aherbst-broad are you thinking we do this when the API to do this when we combine the APIs to create a workspace?